extension/content.js

```diff
@@ -277,6 +277,22 @@ function addDiffViewWithoutWhitespaceOption(type) {
 	}
 }
 
+function addOPLabels(type) {
+	const label = `
+		<span class="timeline-comment-label tooltipped tooltipped-multiline tooltipped-s" aria-label="This user submitted this ${type}.">
+			Original Poster
+		</span>
+	`;
+
+	const comments = $('.js-comment').toArray();
```

Comment:
`const comments = $('div.js-comment').toArray();`

Without the specificity for a `div` this feature doesn't work when someone has the ability to edit another users comments. (The first item in the array is a `form`.)